test/contracts/storage/apollo_deposit_store.js

```diff
@@ -73,6 +73,10 @@ describe('Apollo Deposit Contract', () => {
     await expect(releaseDeposit(from, other, other)).to.be.eventually.rejected;
   });
 
+  it('can not release deposit if refund address equal 0x0', async () => {
+    await expect(releaseDeposit(from, '0x0', from)).to.be.eventually.rejected;
```

Comment:
I'm not actually sure if it fails because address is 0 or because it does not have correct format (40 digits)